### PR TITLE
added name inputs; removed spinner;

### PR DESCRIPTION
**Pull request by CarlRay05**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -237,27 +237,3 @@ hr {
...
-	100% {
-		-webkit-transform: rotate(360deg);
-	}
-}
-@keyframes spin {
-	100% {
-		-webkit-transform: rotate(360deg);
-		transform: rotate(360deg);
-	}
-}
```
Comment (CarlRay05) [inline css/style.css:263]: spinner

**Review by CarlRay05 — COMMENTED** [review]
approve this

**Review by ghost — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -75,16 +75,15 @@ <h3 class="modalHeader">Thanks for playing RAISE IT bingo</h3>
             <div class="modalText">
                 Submit your name and email for a chance to win a Value Integrity Week prize. Thank you for participating.
             </div>
+            <input id="firstName" type="text" placeholder="First name" class="modalInput" ng-class="{'modalDisabled': isSavingEmail}" ng-model="modalData.firstName" />
+            <input type="text" placeholder="Last name" class="modalInput" ng-class="{'modalDisabled': isSavingEmail}" ng-model="modalData.lastName" />
```
Comment (CarlRay05) [inline index.html:79]: <input id="firstName" type="text" placeholder="First name" class="modalInput" ng-class="{'modalDisabled': isSavingEmail}" ng-model="modalData.firstName" />
            <input type="text" placeholder="Last name" class="modalInput" ng-class="{'modalDisabled': isSavingEmail}" ng-model="modalData.lastName" />

File: index.html [hunk 3]
```diff
@@ -75,16 +75,15 @@ <h3 class="modalHeader">Thanks for playing RAISE IT bingo</h3>
...
             <input type="text" placeholder="Email" class="modalInput" ng-class="{'modalDisabled': isSavingEmail}" ng-model="modalData.userEmail" />
             <button type="button" class="modalButton" ng-class="{'modalDisabled': isSavingEmail}" ng-click="submitEmail()">
-                <span ng-show="!isSavingEmail">Submit</span>
-                <img class="spinner" src="images/loader-line.svg" alt="Submitting" ng-show="isSavingEmail" />            
+                Submit
```
Comment (CarlRay05) [inline index.html:82]: remove spinner

File: index.html [hunk 4]
```diff
@@ -75,16 +75,15 @@ <h3 class="modalHeader">Thanks for playing RAISE IT bingo</h3>
...
             </button>
             <div class="modalError">
                 {{modalData.errorMessage}}
-                <div ng-show="isEmailSubmitted && !modalData.userEmail && !modalData.errorMessage">
-                    Enter your DFA email address
-                </div>
+                {{modalFeedbackDisplay()}}
```
Comment (CarlRay05) [inline index.html:86]: {{modalFeedbackDisplay()}}

File: index.html [hunk 5]
```diff
@@ -95,7 +94,7 @@ <h3 class="modalHeader">Your response has been recorded</h3>
             <div class="modalText">
                 Thanks for playing!
                 <div>
-                    <a class="link" href="https://dfamilk.sharepoint.com/sites/EthicsandCompliance/">Go to The Parler</a>
+                    <a class="link" href="https://dfamilk.sharepoint.com/sites/EthicsandCompliance/">More information</a>
```
Comment (CarlRay05) [inline index.html:97]: change this to Go to The Parlor

File: css/style.css [hunk 6]
```diff
@@ -213,15 +213,15 @@ hr {
 	font-size: 18px;
 	width: 228px;
 	margin-right: 15px;
+	margin-bottom: 10px;
```
Comment (CarlRay05) [inline css/style.css:216]: margin bottom add

File: css/style.css [hunk 7]
```diff
@@ -213,15 +213,15 @@ hr {
...
 	letter-spacing: .5px;
 	font-family: Oswald, sans-serif;
 }
 
 .modalError {
 	color: var(--DFA-barn-red);
 	font-size: 16px;
-	margin-top: 5px;
-	margin-left: 5px;
+	margin-left: 7px;
```
Comment (CarlRay05) [inline css/style.css:224]: streamline the margin-left

File: css/style.css [hunk 8]
```diff
@@ -213,15 +213,15 @@ hr {
...
+	margin-bottom: 10px;
 	letter-spacing: .5px;
 	font-family: Oswald, sans-serif;
 }
 
 .modalError {
 	color: var(--DFA-barn-red);
 	font-size: 16px;
-	margin-top: 5px;
-	margin-left: 5px;
```
Comment (CarlRay05) [inline css/style.css:224]: remove margin left

File: css/style.css [hunk 9]
```diff
@@ -213,15 +213,15 @@ hr {
...
 	margin-right: 15px;
+	margin-bottom: 10px;
 	letter-spacing: .5px;
 	font-family: Oswald, sans-serif;
 }
 
 .modalError {
 	color: var(--DFA-barn-red);
 	font-size: 16px;
-	margin-top: 5px;
```
Comment (CarlRay05) [inline css/style.css:223]: remove margin top

File: css/style.css (hunk 1 above)
Comment (CarlRay05) [inline css/style.css:263]: remove spinner

**Pull request closed without merging**